docs/gpu-acceleration.md

```diff
@@ -35,13 +35,36 @@ how to do that on Fedora with `dnf`:
 
   # Install lab (assumes a locally-cloned repo)
   # You can clone the repo if you haven't already done so (either one)
-  # gh repo clone instructlab/instructlab
-  # git clone https://github.com/instructlab/instructlab.git
-  pip3 install ./instructlab/
+  # gh repo clone instructlab/instructlab -- --recurse-submodules
+  # git clone --recurse-submodules https://github.com/instructlab/instructlab.git
+  pip install ./instructlab/
   ```
 
 With Python 3.11 installed, it's time to replace some packages!
 
+### llama-cpp-python backends
+
+Go to the project's GitHub to see
+the [supported backends](https://github.com/abetlen/llama-cpp-python?tab=readme-ov-file#supported-backends).
+
+Whichever backend you choose, you'll see a `pip install` command. First
+you have to purge pip's wheel cache to force a rebuild of llama-cpp-python:
+
+ ```shell
+ pip cache remove llama_cpp_python
+ ```
+
+You'll want to add a few options to ensure it gets installed over the
+existing package, has the desired backend, and the correct version.
+
+```shell
+pip install --force-reinstall llama_cpp_python==0.2.55 -C cmake.args="-DLLAMA_$BACKEND=on"
```

Comment:
I'm guessing this is going to easily get out of sync with our `requirements.txt`. Maybe do something to grab the version from there?

Reply:
There are ways, but they require some extra steps to generate an constraint files from our requirements file. A constraint file is almost the same as a requirements file, but it cannot require optional dependencies. We have `llama_cpp_python[server]`.

My plan is to have this fix for summit and then finish #1072 after summit.

Reply:
Ok I’m fine with trusting you’ll come back around to it 😀

Reply:
I'll be back! </arnold>